cibuildwheel/macos.py

```diff
@@ -187,7 +187,7 @@ def setup_python(python_configuration: PythonConfiguration,
 
 
 def build(options: BuildOptions) -> None:
-    if options.architectures != [Architecture.x86_64]:
+    if options.architectures <= {Architecture.x86_64}:
```

Comment:
You've missed the `not`, I think

Reply:
(oh, this is only 8 minutes old; sorry for the impatience!)

Reply:
Yes, I did. 😳 

Working on mypy at the moment, it should have been able to point out the issues in the tests comparing sets and lists before runtime.

Reply:
MyPy support is kind of a mess. There's a typo in the config so everything that's supposed to be set is not being set. :/ Will have to manually fix for now, then work on fixing the MyPy config. Also, mypy support in "test" is harder, since we run the tests on Python 2 :/.

Reply:
BTW we don't run `test` on Python 2, only the generated projects run on Python 2.